downloads/templatetags/download_tags.py

```diff
@@ -14,3 +14,37 @@ def has_sigstore_materials(files):
         f.sigstore_bundle_file or f.sigstore_cert_file or f.sigstore_signature_file
         for f in files
     )
+
+
+@register.filter
+def sort_windows(files):
+    if not files:
+        return files
+
+    # Put Windows files in preferred order
+    files = list(files)
+    windows_files = []
+    other_files = []
+    for preferred in (
+        'Windows installer (64-bit)',
```

Comment:
Are these strings generated by code elsewhere in the codebase or manually input by Release Managers?

If it is manual input, this feels brittle and likely to cause confusion.

Reply:
They appear to be strings stored on the [ReleaseFile model via the NameSlugModel mixin](https://github.com/python/pythondotorg/blob/c4ee749942227ca75c8e670546afe67232d647b2/downloads/models.py#L305). I'm pretty iffy on handling this this way honestly.

Reply:
@Yh1gs for RM input

Reply:
Yes, they're freeform strings, set by the script adding the ReleaseFile objects: https://github.com/python/release-tools/blob/master/add-to-pydotorg.py#L90.

Reply:
Before checking for the exact strings, I had something that checked for substrings like "(64-bit)", "(32-bit)" and "Windows embeddable package", see the old version at: [`dadd924` (#2311)](https://github.com/python/pythondotorg/pull/2311/commits/dadd9249a3910fd28f992580ad0b92f92495ff75)

Would you prefer something like that? Or some other approach?

Reply:
IMHO having the strings is fine, if comments were added to reference this file from the other and vice-versa